### PR TITLE
[OSDOCS-6239]: CPMS for Nutanix: Support statements

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.14+

Issue:
[OSDOCS-6237](https://issues.redhat.com//browse/OSDOCS-6237) > [OSDOCS-6239](https://issues.redhat.com//browse/OSDOCS-6239)

Link to docs preview:
- [Limitations](https://60568--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-about.html#cpmso-limitations_cpmso-about)
- [Supported cloud providers](https://60568--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-getting-started.html#cpmso-platform-matrix_cpmso-getting-started)
- [Failure domain platform support and configuration](https://60568--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-resiliency.html#cpmso-failure-domains-provider_cpmso-resiliency)
- [Control Plane Machine Set Operator reference](https://60568--docspreview.netlify.app/openshift-enterprise/latest/operators/operator-reference.html#control-plane-machine-set-operator_cluster-operators-ref)

QE review:
- [x] QE has approved this change.

Additional information:
Found a couple of things to tidy in here as well, all minor.

### CODE REVIEW

File: machine_management/control_plane_machine_management/cpmso-getting-started.adoc [hunk 1]
```diff
@@ -43,6 +43,11 @@ The status of the control plane machine set after installation depends on your c
 |X
 |
 
+|Nutanix
+|#TBD#
```
Comment (jeana-redhat) [inline machine_management/control_plane_machine_management/cpmso-getting-started.adoc:47]: [Asked here](https://redhat-internal.slack.com/archives/C0211848DBN/p1685030335536369?thread_ts=1685004438.712299&cid=C0211848DBN)

File: machine_management/control_plane_machine_management/cpmso-getting-started.adoc [hunk 2]
```diff
@@ -54,22 +59,23 @@ The status of the control plane machine set after installation depends on your c
...
+* xref:../../machine_management/control_plane_machine_management/cpmso-configuration.adoc#cpmso-configuration[Control plane machine set configuration]
 
 //Creating a control plane machine set custom resource
 include::modules/cpmso-creating-cr.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../../machine_management/control_plane_machine_management/cpmso-configuration.adoc#cpmso-configuration[Control Plane Machine Set Operator configuration]
+* xref:../../machine_management/control_plane_machine_management/cpmso-using.adoc#cpmso-feat-config-update_cpmso-using[Updating the control plane configuration]
+* xref:../../machine_management/control_plane_machine_management/cpmso-configuration.adoc#cpmso-configuration[Control plane machine set configuration]
```
Comment (jeana-redhat) [inline machine_management/control_plane_machine_management/cpmso-getting-started.adoc:78]: misc link fixes

File: machine_management/index.adoc [hunk 3]
```diff
@@ -64,8 +64,12 @@ As a cluster administrator, you can perform the following actions:
 
 ** xref:../machine_management/control_plane_machine_management/cpmso-configuration.adoc#cpmso-sample-yaml-aws_cpmso-configuration[AWS]
 
+** xref:../machine_management/control_plane_machine_management/cpmso-configuration.adoc#cpmso-sample-yaml-gcp_cpmso-configuration[GCP]
```
Comment (jeana-redhat) [inline machine_management/index.adoc:67]: Missed in 4.13

File: machine_management/index.adoc [hunk 4]
```diff
@@ -64,8 +64,12 @@ As a cluster administrator, you can perform the following actions:
...
+
 ** xref:../machine_management/control_plane_machine_management/cpmso-configuration.adoc#cpmso-sample-yaml-azure_cpmso-configuration[Azure]
 
+//**<YAML xref>[Nutanix]
```
Comment (jeana-redhat) [inline machine_management/index.adoc:71]: To-do in YAML PR

File: modules/cpmso-failure-domains-provider.adoc [hunk 5]
```diff
@@ -21,6 +21,11 @@ The control plane machine set concept of a failure domain is analogous to existi
 |X
 |link:https://cloud.google.com/compute/docs/regions-zones[zone]
 
+|Nutanix
+//link:https://portal.nutanix.com/page/documents/details?targetId=Web-Console-Guide-Prism-v6_1:arc-failure-modes-c.html[Availability domain]
```
Comment (jeana-redhat) [inline modules/cpmso-failure-domains-provider.adoc:25]: Reference for later

**Review by JoelSpeed — COMMENTED** [review]
/lgtm

**Comment by thunderboltsid** [comment]
LGTM

**Comment by huali9** [comment]
/lgtm
Thanks @jeana-redhat

File: modules/cpmso-failure-domains-provider.adoc [hunk 6]
```diff
@@ -21,6 +21,11 @@ The control plane machine set concept of a failure domain is analogous to existi
 |X
 |link:https://cloud.google.com/compute/docs/regions-zones[zone]
```
Comment (dfitzmau) [inline modules/cpmso-failure-domains-provider.adoc:22]: Could be out of scope but this should state `Regions and zones`.

File: modules/cpmso-failure-domains-provider.adoc (hunk 5 above)
Comment (dfitzmau) [reply modules/cpmso-failure-domains-provider.adoc:25]: Should another Jira be raised to ensure that this link gets added to the docs? 

I'm not a fan of comments as they can easily slip through the cracks as reminders.

File: modules/cpmso-failure-domains-provider.adoc [hunk 7]
```diff
@@ -29,5 +34,9 @@ The control plane machine set concept of a failure domain is analogous to existi
 |
 |Not applicable
 |====
+[.small]
+--
+1. Nutanix has a failure domain concept, but {product-title} {product-version} does not include support for this feature.
```
Comment (dfitzmau) [inline modules/cpmso-failure-domains-provider.adoc:39]: Nice!

**Comment by dfitzmau** [comment]
Hi @jeana-redhat . Nice work on the updates. I'll need a senior reviewer to review this content also, because I've only started peer reviews this week. 

I've pinged the B squad.

File: machine_management/control_plane_machine_management/cpmso-about.adoc [hunk 8]
```diff
@@ -25,7 +25,7 @@ The Control Plane Machine Set Operator has the following limitations:
 
 * The Operator requires the Machine API Operator to be operational and is therefore not supported on clusters with manually provisioned machines. When installing a {product-title} cluster with manually provisioned machines for a platform that creates an active generated `ControlPlaneMachineSet` custom resource (CR), you must remove the Kubernetes manifest files that define the control plane machine set as instructed in the installation process.
 
-* Only Amazon Web Services (AWS), Google Cloud Platform (GCP), Microsoft Azure, and VMware vSphere clusters are supported.
```
[the remark this answers is not in the document]
Comment (jeana-redhat) [reply machine_management/control_plane_machine_management/cpmso-about.adoc:28]: Excellent catch!

File: modules/cpmso-failure-domains-provider.adoc (hunk 6 above)
Comment (jeana-redhat) [reply modules/cpmso-failure-domains-provider.adoc:22]: IIRC, we are only referring to zones here, but the GCP docs page has both on one page

**Comment by mburke5678** [comment]
@jeana-redhat Need to squash commits. Otherwise LGTM

FYI @dfitzmau

**Pull request merged**